File: spring/prog_terminal_marittimo/src/main/java/progetto_terminal_marittimo/fornitore.java
package progetto_terminal_marittimo;

public class fornitore {
    private int ID;
    private String username;
    private String password;


    public fornitore(int id, String u, String p){
        this.ID=id;
        this.username=u;
        this.password=p;
    }

    public int getID(){
        return this.ID;
    }

    public String getUsername(){
        return this.username;
    }

    public String getPassword(){
        return this.password;
    }
}

File: pagine/js/fornitore.js
class Fornitore {
    id;
    username;
    password;
    nome;
  
    constructor(id, username, password, n) {
      this.id = id;
      this.username = username;
      this.password = password;
      this.nome=n;
    }
  
    getID() {
      return this.id;
    }
  
    getUsername() {
      return this.username;
    }
  
    getPassword() {
      return this.password;
    }

    getNome() {
      return this.nome;
    }
  
    toString() {
      return "ID: "+this.id+", Username: "+this.username;
    }

    Parse(str){
      let vett=str.split(",");
      this.id=vett[0];
      this.username=vett[1];
      this.password=vett[2];
    }
  }
  